app/views/opendata/agents/nodes/mypage/dataset/my_dataset/resources/show.html.erb

```diff
@@ -1,6 +1,6 @@
 <nav class="menu">
-  <%= link_to :"opendata.labels.edit", "#{@item_url}edit/" %>
-  <%= link_to :"opendata.labels.delete", "#{@item_url}delete/" %>
+  <%= link_to t("opendata.labels.edit"), "#{@item_url}edit/" %>
+  <%= link_to t("opendata.labels.delete"), "#{@item_url}delete/" %>
```

Comment:
__[brakeman]__
* Unsafe parameter value in link_to href [:ref](http://brakemanscanner.org/docs/warning_types/link_to_href)

<!-- Comment by [SideCI](https://sideci.com) -->